website/src/data/members.js

```diff
@@ -32,6 +32,7 @@ import zion from '../images/ourTeam/memberImages/zion.jpg';
 import ethanho from '../images/ourTeam/memberImages/ethanho.jpg';
 import matthewc from '../images/ourTeam/memberImages/matthewc.png';
 import stevenzhou from '../images/ourTeam/memberImages/stevenzhou.png';
+import bensonmanzano from '../image/ourTeam/memberImages/bensonmanzano.png';

```

Comment:
It's looking like Yarn is throwing a compilation error on this line: maybe check the filepath?